localdev/kubechecks/values.yaml

```diff
@@ -24,14 +24,15 @@ configMap:
     # KUBECHECKS_SCHEMAS_LOCATION: https://github.com/zapier/kubecheck-schemas.git
     KUBECHECKS_TIDY_OUTDATED_COMMENTS_MODE: "delete"
     KUBECHECKS_ENABLE_CONFTEST: "false"
+    KUBECHECKS_ENABLE_SHALLOW_CLONE: "true"
 
 
 deployment:
   annotations:
     reloader.stakater.com/auto: "true" 
   
   image:
-    pullPolicy: Never
+    pullPolicy: IfNotPresent
```

Comment:
was this necessary? I thought tilt manages the images internally?

Reply:
What's your setup? I use minikube and tilt-registry -- k8s needs to pull the image from the registry, hence the `IfNotPresent`

Reply:
I tend to use my homelab for tests, so I don't have much experience. I'm good with it if it works for you 👍